api/controllers/build.js

```diff
@@ -3,9 +3,30 @@ const GithubBuildStatusReporter = require('../services/GithubBuildStatusReporter
 const siteAuthorizer = require('../authorizers/site');
 const BuildResolver = require('../services/BuildResolver');
 const { Build, Site } = require('../models');
+const logger = require('winston');
 
 const decodeb64 = str => new Buffer(str, 'base64').toString('utf8');
 
+function emitBuildStatus(socket, build) {
+  try {
+    Site.findById(build.site)
+    .then((site) => {
+      const msg = {
+        id: build.id,
+        state: build.state,
+        site: build.site,
+        branch: build.branch,
+        owner: site.owner,
+        repository: site.repository,
+      };
+      socket.to(build.site).emit('build status', msg);
+      Promise.resolve();
```

Comment:
I don't think we need a Promise.resolve here.

Reply:
thanks ... pls re-review